src/include/duckdb/common/vector_operations/generic_executor.hpp

```diff
@@ -259,7 +284,7 @@ struct GenericExecutor {
 				continue;
 			}
 			A_TYPE input;
-			if (!A_TYPE::ConstructType(state, i, input)) {
+			if (!A_TYPE::ConstructType(state, i, false, input)) {
```

Comment:
Maybe instead of having `ConstructType` return false if any child is `NULL`, we can have a method `ContainsNull()` on the type, so we could do this instead:

```cpp
A_TYPE input = A_TYPE::ConstructType(state, i);
if (input.ContainsNull()) {
     FlatVector::SetNull(result, i, true);
     continue;
}
```

Then we can also remove the validity check from a few lines up.